libs/community/langchain_community/utilities/pebblo.py

```diff
@@ -159,7 +176,7 @@ def get_loader_type(loader: str) -> str:
     for loader_type, loaders in LOADER_TYPE_MAPPING.items():
         if loader in loaders:
             return loader_type
-    return "unknown"
+    return "unsupported"
```

Comment:
I don't know the API, but return a string type on an unkown loader feels like the wrong pattern -- should probably raise or return a None. Anyway going to merge as is feel free to update the implementation if you want in a separate PR

Reply:
Thank you for your feedback @eyurtsev. We appreciate your input on the pattern used for handling unknown loaders. Just to clarify, Pebblo is owned by our company Daxa.ai, and it is designed to support all loader types.

The reason for returning a string instead of raising an error or returning None for unknown loaders is to ensure that users are not restricted from using Pebblo with loaders that have not been explicitly tested by us. By silently returning a string, we aim to provide a seamless experience for users while still indicating that the loader may not be officially supported.

Reply:
we understand your concerns and will take them into consideration for future updates.